config/prow/cluster/cerberus_configmaps.yaml

```diff
@@ -0,0 +1,656 @@
+
+        # Parse and read the config
+        if os.path.isfile(cfg):
+            with open(cfg, 'r') as f:
+                config = yaml.full_load(f)
+            distribution = config["cerberus"].get("distribution", "openshift").lower()
+            kubeconfig_path = config["cerberus"].get("kubeconfig_path", "")
+            port = config["cerberus"].get("port", 8080)
+            watch_nodes = config["cerberus"].get("watch_nodes", False)
+            watch_cluster_operators = config["cerberus"].get("watch_cluster_operators", False)
+            watch_namespaces = config["cerberus"].get("watch_namespaces", [])
+            watch_url_routes = config["cerberus"].get("watch_url_routes", [])
+            cerberus_publish_status = config["cerberus"].get("cerberus_publish_status", False)
+            inspect_components = config["cerberus"].get("inspect_components", False)
+            slack_integration = config["cerberus"].get("slack_integration", False)
+            prometheus_url = config["cerberus"].get("prometheus_url", "")
+            prometheus_bearer_token = config["cerberus"].get("prometheus_bearer_token", "")
+            custom_checks = config["cerberus"].get("custom_checks", [])
+            iterations = config["tunings"].get("iterations", 0)
+            sleep_time = config["tunings"].get("sleep_time", 0)
+            request_chunk_size = config["tunings"].get("kube_api_request_chunk_size", 250)
+            daemon_mode = config["tunings"].get("daemon_mode", False)
+            cores_usage_percentage = config["tunings"].get("cores_usage_percentage", 0.5)
+            database_path = config["database"].get("database_path", "/tmp/cerberus.db")
+            reuse_database = config["database"].get("reuse_database", False)
+
+            # Initialize clients and set kube api request chunk size
+            if not os.path.isfile(kubeconfig_path):
+                kubeconfig_path = None
+            logging.info("Initializing client to talk to the Kubernetes cluster")
+            kubecli.initialize_clients(kubeconfig_path, request_chunk_size)
+
+            if "openshift-sdn" in watch_namespaces:
+                sdn_namespace = kubecli.check_sdn_namespace()
+                watch_namespaces = [namespace.replace('openshift-sdn', sdn_namespace)
+                                    for namespace in watch_namespaces]
+
+            # Check if all the namespaces under watch_namespaces are valid
+            #watch_namespaces = kubecli.check_namespaces(watch_namespaces)
+
+            # Cluster info
+            logging.info("Fetching cluster info")
+            if distribution == "openshift":
+                cluster_version = runcommand.invoke("kubectl get clusterversion")
+                logging.info("\n%s" % (cluster_version))
+            #cluster_info = runcommand.invoke("kubectl cluster-info | awk 'NR==1' | sed -r "
+            #                                 "'s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g'")  # noqa
```

Comment:
Commenting this command to invoke `cluster-info` as this needs _role to get services_ resource!